loops.js

```diff
@@ -34,6 +42,16 @@
 // ==========================================
 
 
+for (let i = 0; i <= 15; i++) {
+    if (i % 2 === 0) {
+        console.log(i + " is even");
+    }
+    if (i % 2 === 1) {
```

Comment:
de uitwerkingen schrijven weer:  " if (i % 2 !== 0) { ". maar goed, het gaat er om dat we in Rome aankomen he, via welke weg maakt niet veel uit;)